inspire/base/static/less/base/header.less

```diff
@@ -71,3 +71,110 @@ body {
         padding-top: 51px;
     }
 }
+
+/* Dropdown menu in Header styles
+-------------------------------------------------- */
+.dropdown-menu-arrow:before {
+        position: absolute;
+        top: -7px;
+        left: 173px;
+        display: inline-block;
+        border-right: 7px solid transparent;
+        border-bottom: 7px solid #ccc;
+        border-left: 7px solid transparent;
+        border-bottom-color: rgba(0, 0, 0, 0.2);
+        content: '';
+    }
```

Comment:
there is a lot of indentation and ``}`` should not be indented. Please, leave all the file with 4 spaces.